.github/workflows/main.yml

```diff
@@ -7,7 +7,7 @@ on:
 
 jobs:
   build:
-    name: Python 3.8 | TiDB nightly | Ubuntu latest
```

Comment:
Actually we run three jobs, but they are all inside one job, with title "TiDB nightly". It's not right. Refer https://github.com/pingcap/django-tidb/blob/main/.github/workflows/ci.yml.